apps/api/src/app/workflows-v2/workflow.controller.ts

```diff
@@ -66,25 +70,29 @@ export class WorkflowController {
     private previewUsecase: PreviewUsecase,
     private buildWorkflowTestDataUseCase: BuildWorkflowTestDataUseCase,
     private buildStepDataUsecase: BuildStepDataUsecase,
-    private patchStepDataUsecase: PatchStepUsecase,
     private patchWorkflowUsecase: PatchWorkflowUsecase,
     private duplicateWorkflowUseCase: DuplicateWorkflowUseCase
   ) {}
 
   @Post('')
+  @ExternalApiAccessible()
```

Comment:
Expose the basic CRUD endpoints for API access. We won't document them yet.

Reply:
im not against this, but as far as i remember, we agreed that this route would be internal.
when needed, we would introduce a lean route specifically for api usage.